src/m365/spo/commands/web/WebProperties.ts

```diff
@@ -34,6 +34,7 @@ export interface WebProperties {
   UIVersionConfigurationEnabled: boolean;
   Url: string;
   WebTemplate: string;
+  RoleAssignments?: any;
```

Comment:
could we avoid using `any` and create an Interface for this and for some part we may reuse [RoleDefinition](https://github.com/pnp/cli-microsoft365/blob/main/src/m365/spo/commands/roledefinition/RoleDefinition.ts)  interface extending it with description property as well?

Reply:
That's an excellent idea!